lib/foreman_theme_satellite/branded_words.rb

```diff
@@ -7,48 +7,17 @@ module ForemanThemeSatellite
   # For now, modifiers are not supported by the generator.
   # Make sure the regexp works both with Ruby and JavaScript.
   # Entries are processed in order and first wins.
+  # Occurrences of "proxy" should be treated as a bug as it is unclear whether the author refers to a HTTP(S) proxy or a Capsule
```

Comment:
Turns out this is used a lot. Very imprecise quick look:

```console
$ rg -i 'proxy' {foreman,katello,foreman_*,foreman-tasks}/locale/**.pot | grep msgid | grep -viE '(smart|http) proxy'
foreman_virt_who_configure/locale/foreman_virt_who_configure.pot:msgid "Ignore Proxy"
foreman_virt_who_configure/locale/foreman_virt_who_configure.pot:msgid "Ignore proxy"
foreman_webhooks/locale/foreman_webhooks.pot:msgid "Proxy Authorization"
foreman_puppet/locale/foreman_puppet.pot:msgid "Import puppet classes from puppet proxy"
foreman_puppet/locale/foreman_puppet.pot:msgid "Import puppet classes from puppet proxy for an environment"
foreman_puppet/locale/foreman_puppet.pot:msgid "No proxy selected!"
foreman_puppet/locale/foreman_puppet.pot:msgid "Invalid proxy selected!"
foreman_puppet/locale/foreman_puppet.pot:msgid "Failed to set %{proxy_type} proxy for %{host}."
foreman_puppet/locale/foreman_puppet.pot:msgid "The %{proxy_type} proxy of the selected hosts was set to %{proxy_name}"
foreman_puppet/locale/foreman_puppet.pot:msgid "The %{proxy_type} proxy of the selected hosts was cleared."
foreman_puppet/locale/foreman_puppet.pot:msgid "The %{proxy_type} proxy could not be set for host: %{host_names}."
foreman_puppet/locale/foreman_puppet.pot:msgid_plural "The %{proxy_type} puppet ca proxy could not be set for hosts: %{host_names}"
foreman_puppet/locale/foreman_puppet.pot:msgid "The %{proxy_type} proxy of the selected hosts was set to %{proxy_name}."
foreman_puppet/locale/foreman_puppet.pot:msgid "Can't find a valid Foreman Proxy with a Puppet feature"
foreman_puppet/locale/foreman_puppet.pot:msgid "Can't find a valid Proxy with a Puppet feature"
foreman_remote_execution/locale/foreman_remote_execution.pot:msgid "List of proxy IDs to be used for remote execution"
foreman_remote_execution/locale/foreman_remote_execution.pot:msgid "Error loading data from proxy"
foreman_remote_execution/locale/foreman_remote_execution.pot:msgid "The only applicable proxy %{proxy_names} is down"
foreman_remote_execution/locale/foreman_remote_execution.pot:msgid_plural "All %{count} applicable proxies are down. Tried %{proxy_names}"
foreman_remote_execution/locale/foreman_remote_execution.pot:msgid "Internal proxy selector can only be used if Katello is enabled"
foreman_remote_execution/locale/foreman_remote_execution.pot:msgid "default_capsule method missing from SmartProxy"
foreman_remote_execution/locale/foreman_remote_execution.pot:msgid "Fallback to Any Proxy"
foreman_remote_execution/locale/foreman_remote_execution.pot:msgid "Enable Global Proxy"
foreman_ansible/locale/foreman_ansible.pot:msgid "Proxy not found"
foreman_ansible/locale/foreman_ansible.pot:msgid "Proxy tasks batch size for Ansible"
foreman-tasks/locale/foreman_tasks.pot:msgid "Send data to the task from external executor (such as smart_proxy_dynflow)"
foreman-tasks/locale/foreman_tasks.pot:msgid "No proxy defined for execution"
foreman-tasks/locale/foreman_tasks.pot:msgid "Failed to check on tasks on proxy at %{url}: %{exception}"
foreman-tasks/locale/foreman_tasks.pot:msgid "Cancel enforced: the task might be still running on the proxy"
foreman-tasks/locale/foreman_tasks.pot:msgid "Task aborted: the task might be still running on the proxy"
foreman-tasks/locale/foreman_tasks.pot:msgid "Proxy action retry count"
foreman-tasks/locale/foreman_tasks.pot:msgid "Proxy action retry interval"
foreman-tasks/locale/foreman_tasks.pot:msgid "Allow proxy batch tasks"
foreman-tasks/locale/foreman_tasks.pot:msgid "Proxy tasks batch size"
foreman_salt/locale/foreman_salt.pot:msgid "ID of Salt Proxy"
foreman_discovery/locale/foreman_discovery.pot:msgid "not required if using a subnet with DHCP proxy"
foreman_discovery/locale/foreman_discovery.pot:msgid "Could not get facts from proxy %{url}: %{error}"
foreman_discovery/locale/foreman_discovery.pot:msgid "Discovery Proxy"
foreman_openscap/locale/foreman_openscap.pot:msgid "No proxy found for %{name} or %{url}"
foreman_openscap/locale/foreman_openscap.pot:msgid "Updated hosts: Assigned with OpenSCAP Proxy: %s"
foreman_openscap/locale/foreman_openscap.pot:msgid "No OpenSCAP Proxy selected."
foreman_openscap/locale/foreman_openscap.pot:msgid "No proxy found!"
foreman_openscap/locale/foreman_openscap.pot:msgid "Change OpenSCAP Proxy"
foreman_openscap/locale/foreman_openscap.pot:msgid "Request timed out. Please try increasing Settings -> proxy_request_timeout"
foreman_openscap/locale/foreman_openscap.pot:msgid "No OpenSCAP proxy found for %{class} with id %{id}"
foreman_openscap/locale/foreman_openscap.pot:msgid "Cannot generate HTML guide, no valid OpenSCAP proxy server found."
foreman_openscap/locale/foreman_openscap.pot:msgid "No proxy with OpenSCAP feature was found."
foreman_openscap/locale/foreman_openscap.pot:msgid "No proxy with OpenSCAP feature is running."
foreman_openscap/locale/foreman_openscap.pot:msgid "No available proxy to validate. Returned with error: %s"
foreman_openscap/locale/foreman_openscap.pot:msgid "Openscap Proxy"
foreman_openscap/locale/foreman_openscap.pot:msgid "Select OpenSCAP Proxy"
foreman_openscap/locale/foreman_openscap.pot:msgid "%s spool errors detected, inspect the appropriate file directly on proxy"
foreman_openscap/locale/foreman_openscap.pot:msgid "Failed to fetch spool status from proxy"
foreman_openscap/locale/foreman_openscap.pot:msgid "OpenSCAP Proxy"
foreman_openscap/locale/foreman_openscap.pot:msgid "ID of OpenSCAP Proxy"
foreman/locale/foreman.pot:msgid "Invalid smart-proxy id"
foreman/locale/foreman.pot:msgid "not required if using a subnet with DHCP proxy"
foreman/locale/foreman.pot:msgid "IPv4 address, not required if using a subnet with DHCP proxy"
foreman/locale/foreman.pot:msgid "IPv6 address, not required if using a subnet with DHCP proxy"
foreman/locale/foreman.pot:msgid "No proxy selected!"
foreman/locale/foreman.pot:msgid "Invalid proxy selected!"
foreman/locale/foreman.pot:msgid "Failed to set %{proxy_type} proxy for %{host}."
foreman/locale/foreman.pot:msgid "The %{proxy_type} proxy of the selected hosts was set to %{proxy_name}"
foreman/locale/foreman.pot:msgid "The %{proxy_type} proxy of the selected hosts was cleared."
foreman/locale/foreman.pot:msgid "The %{proxy_type} proxy could not be set for host: %{host_names}."
foreman/locale/foreman.pot:msgid_plural "The %{proxy_type} puppet ca proxy could not be set for hosts: %{host_names}"
foreman/locale/foreman.pot:msgid "The %{proxy_type} proxy of the selected hosts was set to %{proxy_name}."
foreman/locale/foreman.pot:msgid "Proxy lacks one of the following features: 'Registration', 'Templates'"
foreman/locale/foreman.pot:msgid "The %{proxy_type} proxy of the selected hosts was cleared"
foreman/locale/foreman.pot:msgid "Select desired %s proxy"
foreman/locale/foreman.pot:msgid "*Clear %s proxy*"
foreman/locale/foreman.pot:msgid "Puppet Proxy"
foreman/locale/foreman.pot:msgid "Puppet proxy ID"
foreman/locale/foreman.pot:msgid "Puppet CA Proxy"
foreman/locale/foreman.pot:msgid "Puppet CA proxy ID"
foreman/locale/foreman.pot:msgid "Failed to initialize the PuppetCA proxy: %s"
foreman/locale/foreman.pot:msgid "Failed to initialize the realm proxy: %s"
foreman/locale/foreman.pot:msgid "Realm proxy did not return a one-time password"
foreman/locale/foreman.pot:msgid "DNS Proxy"
foreman/locale/foreman.pot:msgid "DNS proxy ID to use within this domain"
foreman/locale/foreman.pot:msgid "Unable to find a proxy with BMC feature"
foreman/locale/foreman.pot:msgid "HTTP UEFI boot requires proxy with httpboot feature"
foreman/locale/foreman.pot:msgid "Realm proxy"
foreman/locale/foreman.pot:msgid "Realm proxy to use within this realm"
foreman/locale/foreman.pot:msgid "Proxy ID to use within this realm"
foreman/locale/foreman.pot:msgid "Only one declaration of a proxy is allowed"
foreman/locale/foreman.pot:msgid "Unable to communicate with the proxy: %s"
foreman/locale/foreman.pot:msgid "HTTP boot requires proxy with httpboot feature and http_port exposed setting"
foreman/locale/foreman.pot:msgid "HTTPS boot requires proxy with httpboot feature and https_port exposed setting"
foreman/locale/foreman.pot:msgid "Please check the proxy is configured and running on the host."
foreman/locale/foreman.pot:msgid "DHCP Proxy"
foreman/locale/foreman.pot:msgid "DHCP Proxy to use within this subnet"
foreman/locale/foreman.pot:msgid "DHCP Proxy ID to use within this subnet"
foreman/locale/foreman.pot:msgid "TFTP Proxy"
foreman/locale/foreman.pot:msgid "TFTP Proxy ID to use within this subnet"
foreman/locale/foreman.pot:msgid "TFTP Proxy to use within this subnet"
foreman/locale/foreman.pot:msgid "HTTPBoot Proxy"
foreman/locale/foreman.pot:msgid "HTTPBoot Proxy ID to use within this subnet"
foreman/locale/foreman.pot:msgid "HTTPBoot Proxy to use within this subnet"
foreman/locale/foreman.pot:msgid "External IPAM Proxy ID to use within this subnet"
foreman/locale/foreman.pot:msgid "External IPAM Proxy to use within this subnet"
foreman/locale/foreman.pot:msgid "Reverse DNS Proxy"
foreman/locale/foreman.pot:msgid "DNS Proxy ID to use within this subnet"
foreman/locale/foreman.pot:msgid "Template Proxy"
foreman/locale/foreman.pot:msgid "Template HTTP(S) Proxy ID to use within this subnet"
foreman/locale/foreman.pot:msgid "BMC Proxy"
foreman/locale/foreman.pot:msgid "BMC Proxy ID to use within this subnet"
foreman/locale/foreman.pot:msgid "BMC Proxy to use within this subnet for management access"
foreman/locale/foreman.pot:msgid "HTTP(S) proxy"
foreman/locale/foreman.pot:msgid "HTTP(S) proxy except hosts"
foreman/locale/foreman.pot:msgid "Error connecting to %{proxy}: %{error}."
foreman/locale/foreman.pot:msgid "Failed to fetch a free IP from proxy %{proxy}: %{message}"
foreman/locale/foreman.pot:msgid "for proxy"
foreman/locale/foreman.pot:msgid "Proxy reference should be { :relation => [:column_name, ...] }"
foreman/locale/foreman.pot:msgid "Unable to initialize ProxyAPI class %s"
foreman/locale/foreman.pot:msgid "URL of the proxy including schema (https://proxy.example.com:8080)"
foreman/locale/foreman.pot:msgid "SmartProxy|Name"
foreman/locale/foreman.pot:msgid "Create Proxy"
foreman/locale/foreman.pot:msgid "HttpProxy|Name"
foreman/locale/foreman.pot:msgid "HttpProxy|Password"
foreman/locale/foreman.pot:msgid "HttpProxy|Url"
foreman/locale/foreman.pot:msgid "HttpProxy|Username"
foreman/locale/foreman.pot:msgid "SmartProxy|Expired logs"
foreman/locale/foreman.pot:msgid "SmartProxy|Url"
foreman/locale/foreman.pot:msgid "SmartProxyFeature|Capabilities"
foreman/locale/foreman.pot:msgid "SmartProxyFeature|Settings"
katello/locale/katello.pot:msgid "Proxy sync failure"
katello/locale/katello.pot:msgid "A notification about failed proxy sync"
katello/locale/katello.pot:msgid "Prefer registered through proxy for remote execution"
katello/locale/katello.pot:msgid "Prefer using a proxy to which a host is registered when using remote execution"
```

Reply:
I think it looks worse than it actually is...
Either way, we will treat those as bugs and fix them.